release_notes/topics/17_0_0.adoc

```diff
@@ -0,0 +1,6 @@
+= Highlights
+
+== Improved User Search
+
+{project_name} now supports a glob-like syntax for the user search when listing users in admin console, which allows for three different types of searches: prefix (`foo*` which became default search), infix (`*foo*`), and exact `"foo"`)
```

Comment:
in "the Admin Console"  -- we are now standardizing on initial capital letters.
add "the" before "default search"

Reply:
Done.